scripts/components/Sidebar.js

```diff
@@ -29,9 +29,10 @@ export default class Sidebar extends Component {
                 "list-group-item",
                 params.name === name ? "active" : "",
               ].join(" ");
+              const label = collections[name].name || name;
               return (
                 <Link key={i} to={`/collections/${name}`} className={classes}>
-                  {name + (!collections[name].synced ? "*" : "")}
+                  {label + (!collections[name].synced ? "*" : "")}
```

Comment:
Does this still work with that change: https://github.com/mozilla-services/amo-blocklist-ui/commit/218b9bf8e4ebc300b5227eb5edf9e66fee8bf3bb